packages/tools/devtools/devtools-browser-extension/src/popup/InitializePopupView.ts

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import React from "react";
+import ReactDOM from "react-dom";
+import { PopupView } from "./PopupView";
+
+/**
+ * Renders the Fluid Popup view into the provided target element.
+ *
+ * @param target - The element into which the devtools view will be rendered.
+ */
+export async function initializePopupView(target: HTMLElement): Promise<void> {
+	ReactDOM.render(React.createElement(PopupView), target, () => {
+		console.log("Rendered Popup view in devtools window!");
```

Comment:
Nit: this doesn't really render in a devtools "window". Maybe just "Rendered Devtools popup!"?